### PR TITLE
Nica reklama uzduotis

### PR DESCRIPTION
**Pull request by Detaildz**
Bandziau daryt su boostrapu, bet lengviau ir labiau aisku su saasu. 
bootstrapa naudosiu labiau buttonams, prenumaretos langui ir tt.

### CODE REVIEW

File: Bootstrap_task/index.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+              <a class="nav-link" href="#">Straipsniai</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Kontaktai</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div class="mainPage">
```
Comment (pavelVCS) [inline Bootstrap_task/index.html:56]: if you are using BEM methodology, would be nice to keep it everywhere and do not mix with camel case

File: Bootstrap_task/styles/_nav.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.navbar {
+  padding-top: 19px;
+  padding-bottom: 19px;
+  .navbar-brand {
```
Comment (pavelVCS) [inline Bootstrap_task/styles/_nav.scss:4]: you could use "&" instead writing the whole class name:
```
.navbar {
   ....
    &-brand {
        ....
    }
}
```

**Review by pavelVCS — APPROVED** [review]

File: Bootstrap_task/index.html (hunk 1 above)
Comment (Detaildz) [reply Bootstrap_task/index.html:56]: Started page by tutorial video. so made same as on the video. After  that made everything by myself, thats why we came everything mixed. it was planned to change later.

File: Bootstrap_task/styles/_nav.scss (hunk 2 above)
Comment (Detaildz) [reply Bootstrap_task/styles/_nav.scss:4]: oh, i thought we can use & for class if its navbar__brand, not navbar-brand. Thanks

**Pull request merged**